tests/decorators/test_validation.py

```diff
@@ -193,22 +193,3 @@ def test_writeonly_required_error():
     }
     with pytest.raises(ValidationError):
         Draft4RequestValidator(schema).validate({"bar": "baz"})
-
-
-def test_formdata_extra_parameter_strict():
```

Comment:
Formdata is no longer validated with the `ParameterValidator`. We still have API tests testing this behavior.

Reply:
We should still check for this behaviour, no? If there are any extra parameters when `strict_validation=True`?

Reply:
We still do here in the API tests:
https://github.com/spec-first/connexion/blob/1ab5400c0b0abe0483181dbbe972d00fa1d6e077/tests/api/test_parameters.py#L267-L276